### PR TITLE
Add custom list renderer and multi image select to the stories

### PR DESCRIPTION
**Pull request by wamujlb**
Fixes #
Added custom listRenderer and selectedItemHeight props

## Proposed Changes

  - With custom `listRenderer` now we can create multiImageSelect carousel (for carousel I have used react-slick)
  - `selectedItemHeight` by default equal `itemHeight`
  - Added custom listRenderer to the stories 
  - Added test:updateSnapshot command

listrenderergif

### CODE REVIEW

**Comment by liorheber** [comment]
Hi @wamujlb ,
We really appreciate your contribution to the project! You change looks like a great use case!

Could I please ask that you split your changes so that we have one PR per feature?
Additionally, please note that react-slick is used only for the story and therefore should be in the devDependencies.

Thanks again and I really look forward to seeing your PRs.

**Comment by wamujlb** [comment]
@liorheber ,

Thank you. There is only 1 feature `listRenderer`. Multiple image select is just a demo for it. 

About the `react-slick` I will add it to `dev`. No problem :)

File: src/components/multi_select.js [hunk 1]
```diff
@@ -100,7 +102,9 @@ export class MultiSelect extends PureComponent {
       maxSelectedItems,
       searchValue,
       withGrouping,
-      generateClassName
+      generateClassName,
+      listRenderer,
+      selectedItemHeight
```
Comment (liorheber) [inline src/components/multi_select.js]: This was the "additional feature" :)

Could you also make sure that the change is reflected in the Readme file to ensure that users know how to use it properly?
Mainly we should expose the interface expected of listRenderer to work.

Comment (wamujlb) [reply src/components/multi_select.js]: Okay. I can separate this for 2 PRs. `selectedItemHeight` also needed to implement that Multiple Image select.

Comment (wamujlb) [reply src/components/multi_select.js]: @liorheber resolved your comments. I will create new PR for the `selectedItemHeight`

**Comment by liorheber** [comment]
Thanks for your contribution!
Hope to see more in the future :)

**Pull request merged**